app/lib/components/tab.js

```diff
@@ -19,6 +20,14 @@ export default class Tab extends Component {
     this.setState({ hovered: false });
   }
 
+  handleClick (event) {
+    if (event.nativeEvent.which === 1) {
+      this.props.isActive ? null : this.props.onSelect();
+    } else if (event.nativeEvent.which === 2) {
```

Comment:
Same with the other variable `const isLeftClick = event.nativeEvent.which === 1`